integrationm/common/com/cultofbits/customizations/audit/AuditInstanceUpdater.groovy

```diff
@@ -93,7 +93,7 @@ class AuditInstanceUpdater {
             }
 
             if (auditField.args == "uri") {
-                updates << [(auditField.name): usermActionPack.getUser(msg.user).data._links.self]
+                updates << [(auditField.name): usermActionPack.getUser(msg.user).getBody(User.class)._links.self]
```

Comment:
You don't need the `User.class`

Reply:
Sem o User.class o getBody está a retornar null.
Conclusão: **User.class** é necessário.